src/client/java/teammates/client/scripts/util/DataMigrationForSanitizedDataHelper.java

```diff
@@ -0,0 +1,40 @@
+package teammates.client.scripts.util;
+
+/**
+ * Utility class that provides methods to help with migrating data to its desanitized form.
+ * Contains a private constructor to prevent instantiation of the class.
+ */
+public final class DataMigrationForSanitizedDataHelper {
+
+    private DataMigrationForSanitizedDataHelper() {
+        // utility class
```

Comment:
`// empty constructor to prevent instantiation because this is a utility class` ?